radix-engine/src/blueprints/identity/package.rs

```diff
@@ -221,34 +227,15 @@ impl IdentityBlueprint {
         Ok((address, bucket))
     }
 
-    pub fn create_ecdsa_virtual<Y>(
-        id: [u8; NodeId::UUID_LENGTH],
+    pub fn create_virtual<Y>(
```

Comment:
As mentioned can we revert this or to clean it up create a separate internal function which combines the two.